cluster/log-dump/logexporter-daemonset.yaml

```diff
@@ -67,6 +67,8 @@ spec:
           requests:
             cpu: 10m
             memory: 10Mi
+      nodeSelector:
+        topology.kubernetes.io/zone: {{.Zone}}
```

Comment:
Wouldn't be cleaner to have
```
nodeSelector:
  {{.NodeSelector}}
```

And then in the log-dump.sh just 
```

if [[ -n "${compute_zone}" ]]; then`
  node_selector="topology.kubernetes.io/zone: ${compute_zone}"
fi
  sed -i'' -e "s@{{.NodeSelector}}@${{node_selector:-}@g" "${KUBE_ROOT}/cluster/log-dump/logexporter-daemonset.yaml"
```

?

Reply:
Done.